plugin/trino-hive-hadoop2/bin/common.sh

```diff
@@ -211,3 +211,14 @@ function deploy_core_site_xml() {
         'sed "${@:2}" "/docker/files/$1" > /etc/hadoop/conf/core-site.xml' \
         bash "$template" "${args[@]}"
 }
+
+# Checks if Gitflow Incremental Builder (GIB) is enabled and the trino-hive-hadoop2 module should be build and/or tested
+function abort_if_not_gib_impacted() {
+    local module=trino-hive-hadoop2
```

Comment:
nit: Maybe use full path to module and change grep to use extended regex and anchor the pattern to beginning and end of line.

Obviously trino-hive-hadoop2 doesn't match multiple modules today but anyone copying this in future would introduce bugs when module name is an infix of some other module name.

Reply:
Done!